token.go

```diff
@@ -7,6 +7,14 @@ import (
 	"time"
 )
 
+
+// DecodePaddingAllowed will switch the codec used for encoding/decoding JWTs respectively. Note that the JWS RFC7515
```

Comment:
It is no used for decoding, not encoding

Reply:
done